x/auction/simulation/operations/msg.go

```diff
@@ -0,0 +1,180 @@
+package operations
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+	"math/rand"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	authexported "github.com/cosmos/cosmos-sdk/x/auth/exported"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+
+	"github.com/kava-labs/kava/x/auction"
+	"github.com/kava-labs/kava/x/auction/keeper"
```

Comment:
nit: you could use only `auction` methods rather than import the keeper and types